### PR TITLE
chore(e2e): Tests for deletion undo

### PR DESCRIPTION
**Pull request by ZaymonFC**

### CODE REVIEW

File: packages/apps/composer-app/src/playwright/collections.spec.ts [hunk 1]
```diff
@@ -91,5 +91,31 @@ test.describe('Collection tests', () => {
       await host.toggleCollectionCollapsed(0);
       await expect(host.getObjectLinks()).toHaveCount(2);
     });
+
+    test('deletion undo restores collection', async () => {
+      await host.closeToast();
```
Comment (wittjosiah) [inline packages/apps/composer-app/src/playwright/collections.spec.ts:96]: this won't work in CI, need to just figure out why you're needing to do this locally? what's your `DX_ENVIRONMENT` set to?

Comment (ZaymonFC) [reply packages/apps/composer-app/src/playwright/collections.spec.ts:96]: Hmm, for some reason my `DX_ENVIRONMENT` is my computer name.

Comment (wittjosiah) [reply packages/apps/composer-app/src/playwright/collections.spec.ts:96]: yup, that's what mine is as well. Does it not end in `.local` or `.lan` mine automatically does, hence the notify condition in the observability plugin being based on that

Comment (ZaymonFC) [reply packages/apps/composer-app/src/playwright/collections.spec.ts:96]: It's `zan.mbp`. I'll add an export!

**Pull request merged**